apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -102,20 +101,32 @@ patron() ->
 %%% API
 %%%=============================================================================
 
+init_per_suite(NodesList, CTConfig) ->
+    init_per_suite(NodesList, #{}, CTConfig).
+
+init_per_suite(NodesList, CustomNodeConfig, CTConfig) ->
+    init_per_suite(NodesList, CustomNodeConfig, [], CTConfig).
+
+init_per_suite(NodesList, CustomNodeCfg, NodeCfgOpts, CTConfig) ->
```

Comment:
I re-assigned the PR to myself while I re-consider whether `init_per_suite` is the appropriate name for this function.

Reply:
Seems good enough.